model-optimizer/unit_tests/mo/utils/utils_test.py

```diff
@@ -1,10 +1,12 @@
 # Copyright (C) 2018-2021 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+import json
 import unittest
 
 import numpy as np
 
+from mo.utils.error import Error
```

Comment:
why are these new imports added? Seems there is no need in this update. Please remove it